src/highrl/envs/env_generator.py

```diff
@@ -244,8 +246,14 @@ def main() -> None:
     small_obs = EnvGeneratorPolicy.small_obs
     env = GeneratorEnv(hard_obs + med_obs + small_obs + 1)
     model = PPO(EnvGeneratorActorCritic, env, verbose=1, device="cuda")
-    model.learn(5000)
-    model.save("generator_model/")
+    teacher_save_model_callback = t_callback.TeacherSaveModelCallback(
+        train_env=env,
+        save_path=f"model_{time}",
+        save_freq=100,
+    )
+    callback = CallbackList([teacher_save_model_callback])
+    model.learn(total_timesteps=10000, callback=callback)
+    model.save("generator_model")
```

Comment:
You have two main issues here:
(1) In `model.save("generator_model")`, you already added that value `"generator_model"` into the `configs/configurations.py` file, why not use that value?

(2) There ALOT of hardcoded values, please omit that.